plugin-server/src/types.ts

```diff
@@ -142,6 +143,13 @@ export interface PluginsServerConfig extends Record<string, any> {
     PERSON_INFO_CACHE_TTL: number
     KAFKA_HEALTHCHECK_SECONDS: number
     HISTORICAL_EXPORTS_ENABLED: boolean
+    OBJECT_STORAGE_ENABLED: boolean
+    OBJECT_STORAGE_HOST: string
+    OBJECT_STORAGE_PORT: number
```

Comment:
I assume this is coming from the env vars, in which case it will be a string (as will all of these) unless there is a specific conversion.

Reply:
yep, I believe this does the conversion https://github.com/PostHog/posthog/blob/bd1813800126d582517004240c4bfc6e5f333741/plugin-server/src/config/config.ts#L198

Reply:
😅